share/spice/yoga_asanas/yoga_asanas.js

```diff
@@ -0,0 +1,46 @@
+
+function ddg_spice_yoga_asanas(apiResult) {
```

Comment:
Pretentious-I'm-the-reviewer-and-I-want-code-to-be-consistent-nit: Should be called `api_result` instead of `apiResult` :)